openmdao/test_suite/scripts/sellar.py

```diff
@@ -3,7 +3,7 @@
 from openmdao.test_suite.components.sellar import SellarDis1, SellarDis2
 
 class SellarMDAConnect(Group):
-    
```

Comment:
I looked through on my branch, and this Group isn't used in any test. However, it looks like `test_sellar_mda_connect` is supposed to use it, but instead duplicates the code in the class. You could change that over to import this class, and then it would be covered.